pkg/types/clusterconfig/clusterconfig.go

```diff
@@ -501,6 +509,21 @@ func (cc *Config) Validate(awsClient *aws.Client) error {
 		return errors.Wrap(ErrorInstanceTypeNotSupportedInRegion(primaryInstanceType, *cc.Region), InstanceTypeKey)
 	}
 
+	if cc.SSLCertificateARN != nil {
+		exists, err := awsClient.DoesCertificateExist(*cc.SSLCertificateARN)
+		if err != nil {
+			if aerr, ok := errors.CauseOrSelf(err).(awserr.Error); ok {
```

Comment:
you could use `aws.CheckErrCode()`. Maybe it should be renamed to `aws.IsErrCode()` while you're at it?